sdks/go.mod

```diff
@@ -23,33 +23,39 @@ module github.com/apache/beam/sdks/v2
 go 1.16
 
 require (
-	cloud.google.com/go/bigquery v1.17.0
-	cloud.google.com/go/datastore v1.5.0
-	cloud.google.com/go/pubsub v1.11.0-beta.schemas
-	cloud.google.com/go/storage v1.15.0
+	cloud.google.com/go/bigquery v1.28.0
+	cloud.google.com/go/compute v1.5.0 // indirect
+	cloud.google.com/go/datastore v1.6.0
+	cloud.google.com/go/iam v0.2.0 // indirect
+	cloud.google.com/go/pubsub v1.18.0
+	cloud.google.com/go/storage v1.21.0
+	github.com/bketelsen/crypt v0.0.4 // indirect
```

Comment:
I got them with go get when I cloned a fresh master repo. Not sure if all of them are related to this PR. But I guess I encountered errors in previous PRs tests when I didn't push go.mod and go.sum changes.

Reply:
Agreed that it's strange. Unless one is adding a new dependency/import it largely shouldn't show up here... strange. But I'm less worried about additions than I am about unintentional version updates.

Note that with Go Modules, none of this will affect things for users unless they've got a conflicting requirement, at which point Go will pick the maximum of the two. This behavior tends to keep things up to date.  (Then one gets into the issue of breaking changes or bad versions, but I won't go into that here.)